src/tests/libecalc/dto/results/test_simple.py

```diff
@@ -5,7 +5,7 @@
 from libecalc.common.component_info.component_level import ComponentLevel
 from libecalc.common.units import Unit
 from libecalc.dto.base import ComponentType
-from libecalc.dto.result.simple import (
+from libecalc.presentation.simple_result.simple import (
```

Comment:
this test should be moved to tests/libecalc/presentation? I know I havent been consistent and updated tests in general based on the rules we want to adhere to, just to check if you thought of that